src/app/static/src/tests/mocks.ts

```diff
@@ -309,6 +309,7 @@ export const mockModelResultResponse = (props: Partial<ModelResultResponse> = {}
             age_group: "1",
             calendar_quarter: "1",
             indicator_id: 1,
+            indicator: 'mock',
```

Comment:
Seemed to need this property although the value 'mock' I picked at random. Still works though.